os.path/os/path.py

```diff
@@ -1,7 +1,38 @@
 import os
-
+import sys
 
 sep = "/"
+altsep = "\\"
+if (sys.platform == 'uefi') and (sys.implementation.name == 'micropython'):
+ sep = '\\'
```

Comment:
This is actually also true for windows. Does this mean all the uefi stuff here is appliccable for windows as well?

Reply:
Possibly; however, I don't have a specific need for windows at the moment? Is there a developer who wants to take on windows?

Reply:
I almost exclusively use Windows so if there's a major effort towards getting this path problem fixed it would be nice and I could contribute. That's also why I think the 'if platform is X and something else is Y' should be abstracted away in one place instead of repeating. However: in the end I just resolved to changing all backslahes into forward slashes whenevr I use paths: IIRC there are other places apart from os.path and fnmatch which have problems with backslashes.